### DISCUSSION TITLE
Does SlickGrid have a plan to develop a Vue version?

### DISCUSSION THREAD

**Discussion by derrick-null** [6pac/SlickGrid#691] [General]
Our legacy product used SlickGrid for performance purposes and works pretty well on IE11. Now we have new products developed with Element UI based on Vue, and it performed well on Chrome. However, when it comes to the IE11, it gets sucked.

The situation we are dealing with is the following:
More than 20 columns in the grid
Some columns like name and id need to be frozen when scrolling
Some columns have nested items like input, select
Some columns need to show content when hovering

On top of these requirements, when the data size reached 50, the grid started to get slow for scrolling, clicking and input, showing the hovered div, and when the data size came to 100, everything was unacceptable.

I hope we can have the Vue version SlickGrid!

**Comment by ghiscoding** [comment]
not directly but there's this old package made by someone else, [vue-slimgrid](https://www.npmjs.com/package/vue-slimgrid), or you could also try to use [Slickgrid-Universal](https://github.com/ghiscoding/slickgrid-universal) which I support and is a wrapper on top of SlickGrid, it's probably usable in Vue by simply importing it, at least someone did that in React so I guess it should also work in Vue too.

**Comment by 6pac** [comment]
I would suspect that there is some fairly straightforward coding problem causing these issues - probably in the data source side of things.  We have users who have fed billions of rows into the grid. The main issue with IE11 (I have to support it too) is the lack of ES6 support.

**Comment by ghiscoding** [comment]
Better late than never, I created [Slickgrid-Vue](https://github.com/ghiscoding/slickgrid-universal/blob/master/frameworks/slickgrid-vue/README.md) which is a wrapper on top of [Slickgrid-Universal](https://github.com/ghiscoding/slickgrid-universal), it requires Vue >=3.4
